### ISSUE TITLE
Hi. Can you post copyrighted materials on mega or telegram?

### ISSUE THREAD

**Issue by muzanjackson** [Ameyanagi/Aozora_bunko_list#1]
Youtube is deleting my comments. Did you find anything worthwhile on that website?

**Comment by Ameyanagi** [comment]
Hi, now I understood what was going on.
I cannot post any copyrighted material, that is a violation of Japanese law. That is why youtube is deleting your comments.
You can do it yourself because you have the right to use them for personal use.

Everything listed here http://slib.net/works/?type=1&length=&copy=99&rating=-13&tag%5B%5D=on is public domain and can be generated. Each story looks relatively short and good.

**Comment by muzanjackson** [comment]
That’s why I’m asking you to post it on telegram or cloud services

**Comment by Ameyanagi** [comment]
I am saying that posting on telegram or any cloud service is not permitted in Japanese law. 
It doesn't mean that it will become legal if you hide it from the public. 

I also don't have any right to access the copyrighted novels that you have because I haven't purchased them. At least I have to buy them, even to generate them for my personal use.

**Comment by muzanjackson** [comment]
Ok, I won’t pressure you. Will you post works from slib.net?

**Comment by Ameyanagi** [comment]
Yes, I will work on it. I need some time to rewrite the code so I can use them for slib.net.
